lib/workers/pr/body/index.ts

```diff
@@ -11,7 +11,7 @@ import { getControls } from './controls';
 import { BranchConfig } from '../../common';
 
 function massageUpdateMetadata(config: BranchConfig): void {
-  config.upgrades.forEach((upgrade) => {
+  config.upgrades.forEach(upgrade => {
```

Comment:
Update your `prettier` (e.g. run `yarn` to update node_modules) and then run `yarn prettier-fix` to revert this change.